### PR TITLE
Update yaml dep for newline fix

### PR DESCRIPTION
**Pull request by Rob0h**
What I Did
------------
- Update yaml dep for newline fix

How I Did it
------------
- Update yaml dep for newline fix

How to verify it
------------
- `ship update` for istio or redis

Description for the Changelog
------------
- Update yaml dep for newline fix

Picture of a Boat (not required but encouraged)
------------
🛶

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -124,3 +124,7 @@ ignored = [
 [[constraint]]
   branch = "master"
   name = "github.com/hashicorp/go-version"
+
+[[override]]
+  name = "github.com/Masterminds/sprig"
+  version = "v2.16.0"
```
Comment (laverya) [inline Gopkg.toml]: Why is this added? It doesn't seem to change the lockfile

Comment (Rob0h) [reply Gopkg.toml]: the first time i ran update it changed the sprig version back to `2.11.0`

Comment (laverya) [reply Gopkg.toml]: And sprig was already updated to v2.16.0 in #737

Comment (laverya) [reply Gopkg.toml]: That's odd...

Comment (laverya) [reply Gopkg.toml]: @Rob0h why `override` then, and not `constraint`?

Overrides are something to be avoided when possible
